### PR TITLE
MM-10550 Updates to system notice

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Updates to the system notice.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-10550

### CODE REVIEW

**Comment by jasonblais** [comment]
@asaadmahmood Attached is the current UI:

image

**Comment by jwilander** [comment]
I'll need to update the snapshot so the build will pass

File: tests/components/__snapshots__/system_notice.test.jsx.snap [hunk 1]
```diff
@@ -23,6 +23,18 @@ exports[`components/SystemNotice should match snapshot for admin, admin notice 1
...
+  <div
+    className="system-notice__info"
+  >
+    <i
+      className="fa fa-eye"
+    />
+    <FormattedMessage
+      defaultMessage="Only visible to System Admins"
+      id="system_notice.adminVisible"
+      values={Object {}}
```
Comment (mkraft) [inline tests/components/__snapshots__/system_notice.test.jsx.snap]: Why the empty object?

Comment (jwilander) [reply tests/components/__snapshots__/system_notice.test.jsx.snap]: I'm not passing anything to it directly, it's likely that the react-intl library we're is using that as a default, or jest/enzyme is filling it in.

File: components/system_notice/notices.jsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ export default [
         title: (
             <FormattedHTMLMessage
                 id='system_notice.title'
-                defaultMessage='<strong>System Message</strong> from Mattermost'
+                defaultMessage='<strong>Notice</strong> from Mattermost'
```
Comment (hmhealey) [inline components/system_notice/notices.jsx]: Should "Mattermost" be the SiteName setting?

Comment (jwilander) [reply components/system_notice/notices.jsx]: Actually that's a really good question. I checked and we do have "Mattermost" hard-coded all over the place so I'm guessing it's OK. @jasonblais thoughts?

Comment (jasonblais) [reply components/system_notice/notices.jsx]: Are there examples of where "Mattermost" is hard-coded? If admins haven't seen "Mattermost", Site Name might make sense here.

Shift probably wouldn't want any strings to be hard-coded for whitelabeling purposes.

Comment (jwilander) [reply components/system_notice/notices.jsx]: Open the translations file (i18n/en.json) and search for "Mattermost", there's a bunch of hits

Comment (jasonblais) [reply components/system_notice/notices.jsx]: Ah, they're hardcoded but in en.json. Then it won't be a problem for Shift.

Also, let's leave it as is. 0/5 if we should use the SiteName more often, but this is more consistent with other strings the admin sees (About Mattermost dialog, System Console, ..)

Comment (jwilander) [reply components/system_notice/notices.jsx]: Yeah I guess "hard-translated" might have been a better phrase :P Anyways sounds good

**Pull request merged**